client/asset/btc/btc.go

```diff
@@ -1302,7 +1297,7 @@ func (btc *ExchangeWallet) spendableUTXOs(confs uint32) ([]*compositeUTXO, map[s
 	utxos := make([]*compositeUTXO, 0, len(unspents))
 	utxoMap := make(map[string]*compositeUTXO, len(unspents))
 	for _, txout := range unspents {
-		if txout.Confirmations >= confs || btc.isDEXChange(txout.TxID, txout.Vout) {
+		if txout.Confirmations >= confs {
```

Comment:
It appears that `Trusted` balance corresponds to `txout.Safe == true`, so we should filter that here.

```
"safe" : xxx              (bool) Whether this output is considered safe to spend. Unconfirmed transactions
                              from outside keys and unconfirmed replacement transactions are considered unsafe
                              and are not eligible for spending by fundrawtransaction and sendtoaddress.
```

Luckily for us, the `safe` field has been in the RPC response since at least 0.16, so we can still get the same utxo set with ltc, just with a little more work. 

On the other hand, redemption transaction outputs are not going to be marked as safe/trusted, so there will be a confirmation wait after redeeming before the funds can be used for trading. We could potentially work around that though.

Reply:
Good find.  I'll filter on Safe.